Makefile.lb

```diff
@@ -0,0 +1,22 @@
+include $(WORKSPACE_TOP)/common/Makefile.env
+
+NVDIMM_UTIL_INSTALL_DIR ?= $(shell component-tool localpath --repo=ndctl --type=$(BUILD_TYPE) nvdimm-util)
+
+NDCTL_DIR := $(WORKSPACE_TOP)/ndctl/
+
+build:
+	cd $(NDCTL_DIR) && ./autogen.sh
+	cd $(NDCTL_DIR) && ./configure CFLAGS='-g -O0' --prefix=$(NVDIMM_UTIL_INSTALL_DIR) --sysconfdir=/etc --libdir=$(NVDIMM_UTIL_INSTALL_DIR)/lib
+	$(Q)$(MAKE)
+	$(Q)$(MAKE) check
```

Comment:
what is make check ?

Reply:
If this is needed, maybe add a link to the original build sequence instructions in a comment

Reply:
yes this is the instructions for the build process for ndctl

Reply:
It's running some kind of unit tests, @oren-lb IMO we can skip this unless we plan to make changes is ndctl

Reply:
I prefer to leave it it part of the instructions suggested by the building instructions and it does not add any significant compilation time.